README.md

```diff
@@ -1 +1,107 @@
-# appointment-diary-front-end
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+

```

Comment:
- [ ] Please, kindly include a screenshot for your frontend app, so you can show off to your users.

That way, your app will be well documented. 💪